client/payment-methods/style.scss

```diff
@@ -60,6 +60,40 @@
 		}
 	}
 
+	&__upe-methods {
+		background: url( 'assets/images/payment-methods/all_local_payments.svg?asset' )
+			no-repeat right;
+		&.background-local-payment-methods {
+			background: url( 'assets/images/payment-methods/local_payments.svg?asset' )
+				no-repeat right;
+		}
+		background-size: auto;
+		margin-right: 10px;
+
+		h3 {
+			font-size: 14px;
+			padding-right: 215px;
+		}
+		p {
+			padding-right: 230px;
+		}
+
+		&-get-started {
+			margin-right: 16px;
+		}
+		&-actions {
+			padding-top: 11px;
+		}
+
+		@media ( max-width: 1024px ) {
```

Comment:
I noticed that the illustration is not hidden in narrow screens when the `.background-local-payment-methods` is active:

<img width="383" alt="Screenshot 2023-06-05 at 11 30 03" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/c041398b-b71e-49db-afea-fdf2be0800ba">

Can you have a look please?

Reply:
Good catch, thank you!

Reply:
Addressed [here](https://github.com/Automattic/woocommerce-payments/pull/6386/commits/76639cff1128d48c4316f7ad67c3fd7458df97b0).

Ready for final review.